org.robocode.Robocode.yml

```diff
@@ -0,0 +1,91 @@
+app-id: org.robocode.Robocode
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: robocode
+rename-appdata-file: robocode.appdata.xml
+rename-desktop-file: robocode.desktop
+rename-icon: robocode
```

Comment:
As they are included in this repo, why not installing them with the right names directly?

Reply:
I prefer it this way as it's more orthogonal. I know it's unlikely we rename the app ID once it's merged in, but I'm all for any automated steps like this one.